qiskit/execute_function.py

```diff
@@ -35,6 +36,13 @@ def _log_submission_time(start_time, end_time):
     logger.info(log_msg)
 
 
+@deprecate_func(
+    additional_msg="This function combines ``transpile`` and ``backend.run``, which is covered "
+    "by ``Sampler`` primitive. Alternatively, you can also run :func:`.transpile` "
```

Comment:
Do you want to add links for the sampler and primitives too?

Reply:
in this context, it is Sampler "as a primitive". Linking it to the reference implementation is an option, but not the only one. Do you suggest to explain that in text?

Reply:
You cold do:
```suggestion
    "by ``Sampler`` :mod:`~qiskit.primitives`. Alternatively, you can also run :func:`.transpile` "
```
Which would link to this page: https://qiskit.org/documentation/apidoc/primitives.html

That being said `BackendSampler` is the closest to what execute does: https://qiskit.org/documentation/stubs/qiskit.primitives.BackendSampler.html#qiskit.primitives.BackendSampler